trace/opencensus/trace/ext/django/middleware.py

```diff
@@ -87,6 +94,35 @@ def __init__(self, get_response=None):
         self._reporter = settings.REPORTER
         self._propagator = settings.PROPAGATOR
 
+        # Initialize the sampler
+        if self._sampler.__name__ == 'FixedRateSampler':
```

Comment:
Are we going to have very similar code for Flask, webapp2, etc?  It seems like it might be better if the sampler could figure this out itself.  Lookups based on __name__ tend to be fragile.  Could we have a generic (not Django-specific) settings object that we pass into the sampler and report constructors that does this lookup?

Reply:
This is just for Django, because we have to do the tracing configuration in the Django middleware. Need to think about better ways to do the config though.